rr_common/launch/sound_node.launch

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<launch>
+    <node pkg="rr_common" type="sound_node" name="sound_node" output="screen">
```

Comment:
`node` tags that do not specify internal tags (ie. `param` tags) should be implemented as a single tag.
This would take the form of `<node pkg="..." type="..." />`